docs/howto/windows.md

```diff
@@ -106,28 +87,132 @@ many of these.)
   but it may be a bit simpler to use the Linux version.
 * One significant limitation of WSL (as of 2018) is that doing a large
   amount of filesystem I/O tends to be slow.  This is a reason to do
-  operations like a Gradle build, perhaps `git` commands, and perhaps
+  operations like a Gradle build ( Gradle is the Java program that does the Android build), perhaps `git` commands, and perhaps
   the occasional `yarn install` using the Windows versions.
 
 [so-guide]: https://stackoverflow.com/questions/42614347/running-react-native-in-wsl-with-the-emulator-running-directly-in-windows
 
-### Accessing the filesystem
-You can access your Windows local drives from `bash` and other Linux
-programs by navigating to `/mnt/<drive-letter>` .
 
-### Running `yarn` commands
-First, install Node.js and Yarn in your Linux environment.  You'll follow
-the same steps as in [our main setup guide](build-run.md) -- just use the
-Linux versions. :-)
+### Step-by-step instructions for setting up WSL
+
+1. Open Powershell as an Administrator and run the command below to enable WSL. 
+
+    `Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux`
+
+2. Install [Chocolatey][chocolatey]. After installing Chocolatey, use it to install the JAVA SE Development Kit with  
+
+    `choco install -y jdk8`
+
+3. Follow the [instructions][android-studio] here to install Android Studio. 
+
+4. In Step 3 above, you saw how to add the `ANDROID_HOME` environment variable. Return to the Environment Variables screen. Scroll down in the User variables section where you added `ANDROID_HOME` until you find a variable called `Path`. 
```

Comment:
Dang, this step is a bit messy.

That is a reason I'm glad to have detailed step-by-step instructions like these!

It also means this is a step where I'm curious to better understand what it's doing (what goes wrong if you leave out one of the substeps here?), and also how you determined this was the set of actions to take to handle that.

Reply:
This is one part of the documentation I really don't like and plan eliminate. I will be taking out each of the substeps and trying to see if it still works and then I can be certain of what's necessary and what's superfluous. Also, I just discovered that Path can be set via the Powershell so I'd probably change that section to include the necessary Powershell commands as these steps are indeed messy.